src/main/kotlin/Card.kt

```diff
@@ -23,15 +23,19 @@ data class Card(val id: Int,
                 organization,
                 repoName,
                 number.toString(),
-                title,
-                description,
+                makePipeSafe(title),
+                makePipeSafe(description),
                 estimate,
-                epicTitle,
+                makePipeSafe(epicTitle),
                 milestone,
                 status,
                 lastUpdateTime,
                 assignees.joinToString(","),
                 labels.joinToString(",")
         ).joinToString("|")
     }
+
+    fun makePipeSafe(value: String): String {
+        return value.replace("|", "[")
+    }
```

Comment:
Not sure why I can't make a suggestion.

If you create an extension function, you could do something like:
```
fun String.makePipeSafe() : String {
    return this.replace("|", "[")
}
```
And then above call `title.makePipeSafe()` instead of `makePipeSafe(title),`

I admit, it's a very low readability gain, but then you could 'pipe' it through other validations. 🤷